src/commands/login.ts

```diff
@@ -24,7 +25,8 @@ Login as HMD successfully!
   async run() {
     const {flags} = this.parse(Login)
 
-    let email = flags.email
+    let email = flags.email || process.env.CMD_EMAIL
```

Comment:
Hi @boneskull Could you please update the environment variable with `CMD_CLI` prefix? I want  it different from [CodiMD's configuration](https://hackmd.io/c/codimd-documentation/%2Fs%2Fcodimd-configuration#Email).